README.md

```diff
@@ -217,11 +192,7 @@ After the initial setup, here’s how to manage your daily development tasks.
 
    This allows you to run tests for a specific parser or package. For running tests across all packages, simply repeat the command for each directory.
 
-7. Making code changes
```

Comment:
This information is good to be documented. If you want to move it out of "Day-to-day tasks", I would suggest to add it as a Note in the "Setup" section.

Reply:
I feel like this information is already reflected in the benefits part at the top of the README, don't you think?

Reply:
I think conveying it more explicitly is better. New developers may be less familiar with editable mode.

Reply:
Moved it to the beginning